src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackNotes/FeedbackNote/FeedbackNote.tsx

```diff
@@ -58,6 +59,29 @@ export const FeedbackNote = ({ note, onEdit, onDelete }: FeedbackNoteProps) => {
 
   return (
     <>
+      <StyledNoteHeader
+        onMouseEnter={() => !isEditMode && setIsVisibleActions(true)}
+        onMouseLeave={() => setIsVisibleActions(false)}
+      >
+        <StyledFlexTopStart>
+          <StyledAuthorLabel color={variables.palette.outline}>
+            {note.user.firstName} {note.user.lastName}
```

Comment:
what if user does not have first name or last name? maybe add some check?

Reply:
these are required fields in refactoring and in legacy as well